packages/wp-source/src/actions.ts

```diff
@@ -53,13 +55,23 @@ const actions: WpSource["actions"]["source"] = {
       // set isHome value if it's true
       if (isHome) source.data[route].isHome = true;
     } catch (e) {
-      console.log(e);
-      // an error happened
-      source.data[route] = {
-        is404: true,
-        isFetching: false,
-        isReady: false
-      };
+      // It's a server error (4xx or 5xx)
+      if (e instanceof ServerError) {
```

Comment:
Maybe I'm wrong but I think that for this check to work this way, you need to change the `api.get`:
``` js
if (!response.ok) {
  const { status, statusText } = response;
  throw { status, statusText, isServerError: true }; // <-- this needs to throw a ServerError
```

Reply:
yes, I had a different data structure before but refactored it to extend from `Error` and must have missed it here.